docs/README.md

```diff
@@ -85,13 +85,13 @@ Please refer to this [page](objects.md) for details on how AKO interprets the Ku
 Please refer to this [page](cc_to_ako.md) for details on how to migrate workloads from cloud connector based Avi controller to AKO based Avi controller.
 
 ### AKO Compatibility Guide
-AKO version 1.5.1 support for Kubernetes, Openshift, Avi Controller is as below:
+AKO version 1.6.1 support for Kubernetes, Openshift, Avi Controller is as below:
 
 | **Orchestrator/ Controller** | **Versions Supported** |
 | --------- | ----------- |
-| `Kubernetes` | 1.16 - 1.21 |
+| `Kubernetes` | 1.19 - 1.22 |
 | `Openshift` | 4.3 - 4.7 |
```

Comment:
openshift starts using k8s 1.19 in 4.6 version, so we would have to update the min version. Openshift 4.9 uses k8s 1.22 which we support, but systest is using openshift 4.8. So we would have to decide the max supported version.

Reply:
addressed